.ci_support/linux_64_.yaml

```
@@ -6,5 +6,7 @@ channel_targets:
 - conda-forge main
 docker_image:
 - quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.9'
```

Need 3.10 as minimal version.
```suggestion
- '3.10'
```